src/data/useGetPages.js

```diff
@@ -0,0 +1,12 @@
+import { mapFolderData } from "../util/mapFolderData";
+import { useAsync } from "../util/useAsync";
+import { axiosInstance } from "../util/axiosInstance";
+
+export const useGetPages = () => {
+  const getUser = () => axiosInstance.get("sample/folder");
```

Comment:
```suggestion
  const getUser = async () => await axiosInstance.get("sample/folder");
```

비동기 로직임을 명시적으로 알 수 있게 async 함수로 작성해주면 좋을듯합니다.